crates/core/migrations/2022-03-24-181306_twitter_handle_name_service_table/up.sql

```diff
@@ -0,0 +1,5 @@
+create table twitter_handle_name_services (
+    address                      varchar(48)    primary key,
+    wallet_address               varchar(48)    not null,
+    twitter_handle               text           not null                      
+);
```

Comment:
We'll want index on wallet_address and twitter_handle.

Reply:
yeah I gotta test this first, I'll add indexes and slot thing to it